### ISSUE TITLE
Followers/following nothing increases

### ISSUE THREAD

**Issue by vishakhag6** [cyberkallan/inshackle-bot#27]
Hi, 
Followers/following nothing increases. On the terminal, the script is returning the same numbers again and again. 
Screenshot:  https://ibb.co/VqLLfNG

**Comment by Unawarding** [comment]
Yep, same here. The bot only follows the same people over and over.
It follows around 30 people, prints "Sleeping 60 secs", unfollows these 30 people, and then follows the same 30 people again.

**Comment by Rgjdhyy** [comment]
Hey that is the technique of following celebgram..and un following

**Comment by nurfauzannajib** [comment]
same here

**Comment by ajeethbt** [comment]
Put any login ID and Password and it starts its shit

**Comment by termsignal** [comment]
The error is coming from the getid file. IG is blocking it sometimes. Yesterday it was fine. 

I think it might blacklist you for a while before it allows you to do it again. testing in progress
